src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -96,6 +97,34 @@ bool IsCKApplicable(const ProblemDescriptionType& problem)
         ptrs.begin(), ptrs.end(), [&args](auto& ptr) { return args.IsSupportedBy(ptr); });
 }
 
+#define WORKAROUND_CK_ISSUE_1184 1
+#ifdef WORKAROUND_CK_ISSUE_1184
+struct HipEventProfiler
+{
+    const Handle& handle;
+    float event_time;
+    HipEventPtr start;
+    HipEventPtr stop;
+
+    HipEventProfiler(const Handle& handle_)
+        : handle(std::move(handle_)),
```

Comment:
I do not think we need std::move here. Or I am missing something?

Reply:
You are right. It is a reference. No need std::move. Fixed.